src/server/logarchiver.cpp

```diff
@@ -37,26 +37,73 @@ namespace KDC {
 
 bool LogArchiver::getLogDirEstimatedSize(uint64_t& size, IoError& ioError) {
     SyncPath logPath = Log::instance()->getLogFilePath().parent_path();
-
+    bool result = false;
     for (int i = 0; i < 2; i++) {  // Retry once in case a log file is archived/created during the first iteration
-        IoHelper::getDirectorySize(logPath, size, ioError);
+        result = IoHelper::getDirectorySize(logPath, size, ioError);
         if (ioError == IoErrorSuccess) {
             return true;
         }
     }
-    return false;
+    LOG_WARN(Log::instance()->getLogger(),
+             "Error in LogArchiver::getLogDirEstimatedSize: " << Utility::formatIoError(logPath, ioError).c_str());
+
+    return result;
 }
 
 ExitCode LogArchiver::generateLogsSupportArchive(bool includeArchivedLogs, const SyncPath& outputPath,
-                                                 const SyncPath& archiveName,
-                                                 ExitCause& exitCause, std::function<void(int)> progressCallback) {
+                                                 std::function<bool(int)> progressCallback, SyncPath& archivePath,
```

Comment:
SyncPath& archivePath => SyncPath &archivePath
ExitCause& exitCause => ExitCause &exitCause

Reply:
Done in commit: [7f8d285](https://github.com/Infomaniak/desktop-kDrive/pull/117/commits/7f8d285f1b93fccd0d693ace3e0d64abbb51cd65)